src/Infra/Infra.Twitch/Extensions/CredentialsExtensions.cs

```diff
@@ -7,9 +7,12 @@ namespace ChatterBot.Infra.Twitch.Extensions
     public static class CredentialsExtensions
     {
         // TODO: #28 Replace CredentialsExtensions with AutoMapper profiles.
-        public static ConnectionCredentials ToTwitchLib(
-            this TwitchCredentials credentials, IDataProtection dataProtection)
-            => new ConnectionCredentials(credentials.Username,
-                dataProtection.Unprotect(credentials.AuthToken).BytesToString());
+        public static ConnectionCredentials ToTwitchLib(this TwitchCredentials credentials, IDataProtection dataProtection)
+        {
+            if (credentials.AuthToken == null)
+                throw new System.ArgumentException("Unable to convert null AuthToken", nameof(credentials));
```

Comment:
Assumed an exception was better here than just letting the decrypt blow up on an empty byte[]

Reply:
Agreed.